test/c/test_elide_permutations.c

```diff
@@ -0,0 +1,115 @@
+// This code is part of Qiskit.
+//
+// (C) Copyright IBM 2025.
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE.txt file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+#include "common.h"
+#include <complex.h>
+#include <math.h>
+#include <qiskit.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <string.h>
+
+/**
+ * Test running the path with no elision.
+ */
+int test_elide_permutations_no_result(void) {
+    const uint32_t num_qubits = 5;
+
+    QkCircuit *qc = qk_circuit_new(5, 0);
```

Comment:
Maybe you meant to use this variable?
```suggestion
    QkCircuit *qc = qk_circuit_new(num_qubits, 0);
```